### ISSUE TITLE
Chip Offset for L1 Tracking

### ISSUE THREAD

**Issue by AgileEngineeringLLC** [pmonta/GNSS-DSP-tools#6]
For offsetting the early and late fingers for L1 tracking, the python tracking code uses 0.05 vs the typical 0.5 (1/2 chip) offset. When i change the 0.05 to 0.5, the EPL values look more acceptable to me, i.e. the P value is always the peak value. Is the 0.05 an error, or is there some reason to use a 1/20 chip offset? This seems way too tight to me. thx!

**Comment by pmonta** [comment]
Yes, the small offset is intentional---it goes by the name "narrow correlator".  It's more resistant to multipath, provided the signal contains several sidelobes of the C/A signal, that is, has double-sided bandwidth at least 6 or 8 MHz.  If something in the system filters the signal to 2 MHz, i.e. just the main lobe, before the correlation step, then the narrow correlator spacing indeed has no advantage.

This is an area (among many) where this software could use improvement.  At the least I should add the "double delta" or "zero spacing" correlator as an option. There are other multipath-resistant schemes under the rubric of "S curve" or "discriminator curve"---see Pany's book, "Navigation Signal Processing for GNSS Software Receivers", for an overview.
